README.md

```diff
@@ -383,60 +384,33 @@ This example shown an ID Token containing a `vp_hash`:
       "kty":"EC",
       "x":"jf3a6dquclZ4PJ0JMU8RuucG9T1O3hpU_S_79sHQi7VZBD9e2VKXPts9lUjaytBm",
       "y":"38VlVE3kNiMEjklFe4Wo4DqdTKkFbK6QrmZf77lCMN2x9bENZoGF2EYFiBsOsnq0"
-   }
+   },
+   "_credential_types":{
+      "https://www.w3.org/2018/credentials/examples/v1/IDCardCredential":[
+         "src1"
+      ]
+   },
+   "_claim_sources":{
+     "src1":{
+         "format":"vp_jwt",
+         "endpoint":"https://op.example.com/presentations/1234564",
```

Comment:
with Distributed claims, vp_ldp would have to be returned as a JWT from the endpoint?

> endpoint
REQUIRED. OAuth 2.0 resource endpoint from which the associated Claim can be retrieved. The endpoint URL MUST return the Claim as a JWT.